app/metrics.yaml

```diff
@@ -382,6 +382,129 @@ search.default_engine:
     - fenix-core@mozilla.com
     expires: "2019-09-01"
 
+bookmarks_management:
+  bookmark_in_new:
+    type: event
+    description: >
+      A user opened a bookmark in a new tab.
+    bugs:
+    - 974
+    data_reviews:
+    - https://github.com/mozilla-mobile/fenix/issues/974
```

Comment:
We should link to the data review (or PR) for these events

Reply:
LOL. Chicken 'n egg problem.